[ONNX] Link to the wiki (#68505)

BowenBao opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #69550
* #69549
* #69548
* #69547
* #69546
* __->__ #69544
* #68491
* #68490

Co-authored-by: Gary Miguel <garymiguel@microsoft.com>

Differential Revision: [D33542418](https://our.internmc.facebook.com/intern/diff/D33542418)

BowenBao committed:
[ONNX] Link to the wiki (#68505)


Co-authored-by: Gary Miguel <garymiguel@microsoft.com>

[ghstack-poisoned]

msaroufim commented:
@msaroufim has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32994274).

BowenBao committed:
Update on "[ONNX] Link to the wiki (#68505)"


Co-authored-by: Gary Miguel <garymiguelmicrosoft.com>

Differential Revision: [D32994274](https://our.internmc.facebook.com/intern/diff/D32994274)

[ghstack-poisoned]

msaroufim commented:
@msaroufim has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32994274).

BowenBao committed:
Update on "[ONNX] Link to the wiki (#68505)"


Co-authored-by: Gary Miguel <garymiguelmicrosoft.com>

[ghstack-poisoned]

msaroufim commented:
@msaroufim has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33542418).
